bundles/org.openhab.binding.openweathermap/src/main/java/org/openhab/binding/openweathermap/internal/handler/OpenWeatherMapAPIHandler.java

```diff
@@ -76,7 +76,8 @@ public void initialize() {
         config = getConfigAs(OpenWeatherMapAPIConfiguration.class);
 
         boolean configValid = true;
-        if (config.apikey.isBlank()) {
+        String apiKey = config.apikey;
```

Comment:
Perhaps you could initialize it to `""` in `OpenWeatherMapAPIConfiguration` and make it `@NonNull`?

Reply:
Yes, also would like to add more null annotations. Will come back to this.

Reply:
Fixed